base/bucket.go

```diff
@@ -51,6 +51,8 @@ const (
 	IndexBucket
 )
 
+const DefaultViewTimeoutSecs = 75 * 1000 // 75s
```

Comment:
Name is misleading when value is actually milliseconds, and will be returning the wrong value in the API response. Maybe keep the const as seconds, and multiply to get milliseconds only when used inside `GetViewQueryTimeoutMs`

```suggestion
const DefaultViewTimeoutSecs = 75
```